staging/src/k8s.io/apiserver/pkg/authentication/authenticatorfactory/delegating.go

```diff
@@ -83,7 +84,7 @@ func (c DelegatingAuthenticatorConfig) New() (authenticator.Request, *spec.Secur
 		if err != nil {
 			return nil, nil, err
 		}
-		cachingTokenAuth := cache.New(tokenAuth, false, c.CacheTTL, c.CacheTTL)
+		cachingTokenAuth := cache.New(context.Background(), tokenAuth, false, c.CacheTTL, c.CacheTTL)
```

Comment:
Consider context.TODO(), since you would change this line if we wanted to make it possible to shut this down?

Reply:
Done.